CTP/common/src/com/navercorp/cubridqa/common/ConfigParameterConstants.java

```diff
@@ -0,0 +1,85 @@
+package com.navercorp.cubridqa.common;
+
+public class ConfigParameterConstants {
+	
+	//Environment variables
+	public static final String SKIP_UPGRADE = "SKIP_UPGRADE";
+	public static final String CTP_BRANCH_NAME = "CTP_BRANCH_NAME";
+	
+	//CUBRID installation and configuration parameters
+	public static final String CUBRID_INSTALL_ROLE = "cubrid_install_role";
+	public static final String CUBRID_DOWNLOAD_URL = "cubrid_download_url";
+	public static final String CUBRID_ADDITIONAL_DOWNLOAD_URL = "cubrid_additional_download_url";
+	public static final String CUBRID_DB_CHARSET = "cubrid_db_charset";
+	public static final String DEFAULT_CUBRID_PROPERTY_PREFIX = "default.cubrid";
```

Comment:
I just added all parameters which are from table of CUBRIDQA-70, so they will be used in ha_repl